argocd/tests/conftest.py

```diff
@@ -50,13 +50,21 @@ def dd_environment(dd_save_state):
             repo_server_host, repo_server_port = stack.enter_context(
                 port_forward(kubeconfig, 'argocd', 8084, 'service', 'argocd-repo-server')
             )
+            notifications_controller_host, notifications_controller_port = stack.enter_context(
+                port_forward(kubeconfig, 'argocd', 9001, 'service', 'argocd-notifications-controller')
```

Comment:
```suggestion
                port_forward(kubeconfig, 'argocd', 9001, 'service', 'argocd-notifications-controller-metrics')
```

Reply:
Seems like that is the correct service that exposes those metrics

Reply:
gotcha - makes sense